src/PointwiseFunctions/AnalyticData/CurvedWaveEquation/RegularSphericalWaveKerrSchild.hpp

```diff
@@ -56,20 +57,30 @@ class RegularSphericalWaveKerrSchild : public MarkAsAnalyticData {
   struct BlackHoleMass {
     using type = double;
     static constexpr OptionString help = {"Mass of the black hole"};
-    static type default_value() noexcept { return 1.; }
+    static type default_value() noexcept {
+      return std::numeric_limits<double>::signaling_NaN();
```

Comment:
no, don't change the defaults in the option structs...  just the default initialization of the member variables...

Reply:
That sounds like an inconsistent thing to do ... unless the `default_value()` is used when the user does not specify any value in the input file (which is different from default constructing the solution). Is that how `default_value()` is used?

Reply:
yes, that is how `default_value()` is used...

Reply:
great, thanks for the explanation. I've changed to NaN's.